apps/web/src/pages/Pools/EditPoolPage/EditPoolPage.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from "react";
 import { defineMessage, useIntl } from "react-intl";
-import RocketLaunchIcon from "@heroicons/react/24/outline/RocketLaunchIcon";
 import ChevronDoubleLeftIcon from "@heroicons/react/24/solid/ChevronDoubleLeftIcon";
+import ExclamationCircleIcon from "@heroicons/react/24/outline/ExclamationCircleIcon";
+import CheckCircleIcon from "@heroicons/react/24/outline/CheckCircleIcon";
+import QuestionMarkCircleIcon from "@heroicons/react/24/outline/QuestionMarkCircleIcon";
```

Comment:
We should probably use the mini versions here. Espcially with how light the colours are they are vibrating for me currently.

```suggestion
import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
import QuestionMarkCircleIcon from "@heroicons/react/20/solid/QuestionMarkCircleIcon";
```